curriculum/challenges/english/25-front-end-development/lab-favorite-icon-toggler/66bf6bacf178eac7b96d4f5e.md

```diff
@@ -131,84 +130,88 @@ spanElements.forEach(span => {
-</head>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
 
-<body>
+  <body>
     <h1>Art Supplies</h1>
     <ul class="item-list">
-        <li>
-            120 gm paper
-            <span class="favorite-icon" id="favoriteIcon1">&#9825;</span>
-        </li>
-        <li>
-            Watercolor set
-            <span class="favorite-icon" id="favoriteIcon2">&#9825;</span>
-        </li>
-        <li>
-            Lead pencil 6B
-            <span class="favorite-icon" id="favoriteIcon3">&#9825;</span>
-        </li>
+      <li>
+        120 gm paper
+        <button class="favorite-icon" id="favoriteIcon1">&#9825;</button>
+      </li>
+      <li>
+        Watercolor set
+        <button class="favorite-icon" id="favoriteIcon2">&#9825;</button>
+      </li>
+      <li>
+        Lead pencil 6B
+        <button class="favorite-icon" id="favoriteIcon3">&#9825;</button>
+      </li>
     </ul>
 
     <script src="script.js"></script>
-</body>
-
+  </body>
 </html>
 ```
 
 ```css
 body {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    height: 100vh;
-    margin: 0;
-    font-family: Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
 }
 
 h1 {
-    margin-bottom: 20px;
+  margin-bottom: 20px;
 }
 
 .item-list {
-    list-style-type: none;
-    padding: 0;
+  list-style-type: none;
+  padding: 0;
 }
 
 .item-list li {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 10px;
-    border-bottom: 1px solid #ddd;
-    width: 200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  border-bottom: 1px solid #ddd;
+  width: 200px;
 }
 
 .favorite-icon {
-    font-size: 20px;
-    cursor: pointer;
+  font-size: 1.25rem;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.filled {
+  color: #d22b2b;
 }
 ```
 
 ```js
-document.addEventListener("DOMContentLoaded", () => {
-  const favoriteIcons = document.querySelectorAll(".favorite-icon");
-
-  favoriteIcons.forEach((icon) => {
-    icon.addEventListener("click", () => {
-      if (icon.classList.contains("filled")) {
-        icon.classList.remove("filled");
-        icon.innerHTML = "&#9825;"; // Empty heart
+document.addEventListener('DOMContentLoaded', () => {
+  const favoriteIcons = document.querySelectorAll('.favorite-icon');
+
+  favoriteIcons.forEach(icon => {
+    icon.addEventListener('click', () => {
+      if (icon.classList.contains('filled')) {
+        icon.classList.remove('filled');
+        icon.innerHTML = '&#9825;'; // Empty heart
       } else {
-        icon.classList.add("filled");
-        icon.innerHTML = "&#10084;"; // Filled black heart
+        icon.classList.add('filled');
+        icon.innerHTML = '&#10084;'; // Filled black heart
```

Comment:
we should stick to double quotes.

https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#formatting-seed-code

Reply:
Hey Jessica, is there a way to use multiple prettier configs so when I try to fix small spacing issues the quotes are kept?

Reply:
I did just fix it though.

Reply:
> is there a way to use multiple prettier configs so when I try to fix small spacing issues the quotes are kept?

this I am not sure about. I personally haven't tried it. You could reach out in the contributors channel to see if anyone has though.